views/pages/login.ejs

```diff
@@ -16,11 +16,11 @@
             <p><%= messages.error%> <%}%></p>
             <form action="/login" method="post">
                 <div>
-                    <label for="email">E-mail adress</label>
+                    <!-- <label for="email">E-mail adress</label> -->
```

Comment:
Zijn de labels ook op de registreer pagina weggehaald? Als iemand nu iets typt kan hij/zij niet meer zien wat de verwachting is van de input

Reply:
Er zijn placeholders in de input, dus de titel van het invoerveld zit in de input zelf